mesheryctl/cmd/helpers.go

```diff
@@ -0,0 +1,14 @@
+package cmd
+
+import (
+	"io"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// SafeClose is a helper function help to close the io
+func SafeClose(co io.Closer) {
```

Comment:
I agree! I was thinking of this since a while. But should we create another issue for it? in which we will pull off all helper functions into a separate module?

Reply:
Yeah, a separate issue would be great too!